### PR TITLE
List of known specs on Detector

### PR DESCRIPTION
**Pull request by wirednkod**

### CODE REVIEW

File: packages/connect/src/Detector.test.ts [hunk 1]
```diff
@@ -18,4 +17,58 @@ describe('Initialize Detector without extension', () => {
...
+  //     expect(api).toBeTruthy();
+  //     done();
+  //   } catch (err: unknown) {
+  //     done(err);
+  //   } finally {
+  //     await detect.disconnect('kusama');
+  //   }
+  // }, 30000);
+
+  test('Should connect with unknown chain westedn2 and chainSpecs.', async done => {
```
Comment (raoulmillais) [inline packages/connect/src/Detector.test.ts:50]: Westend typo

File: packages/connect/src/Detector.test.ts [hunk 2]
```diff
@@ -18,4 +17,58 @@ describe('Initialize Detector without extension', () => {
...
+      expect(api).toBeTruthy();
+      done();
+    } catch (err: unknown) {
+      done(err);
+    } finally {
+      await detect.disconnect('polkadot');
+    }
+  }, 30000);
+
+  // DEACTIVATED for now due to chainSpecs - once Kusama specs are available this should be activated
```
Comment (raoulmillais) [inline packages/connect/src/Detector.test.ts:35]: Add a link to https://github.com/paritytech/devops/issues/767 in the comments

**Review by raoulmillais — CHANGES_REQUESTED** [review]

File: packages/connect/src/Detector.test.ts (hunk 1 above)
Comment (wirednkod) [reply packages/connect/src/Detector.test.ts:50]: It does not matter the typo... It can be whatever else than the known networks

**Comment by wirednkod** [comment]
Refer to https://github.com/paritytech/devops/issues/767 concerning Kusama pending network

File: packages/connect/src/Detector.test.ts (hunk 1 above)
Comment (raoulmillais) [reply packages/connect/src/Detector.test.ts:50]: True but it looks weird 😸

**Review by raoulmillais — APPROVED** [review]

**Pull request merged**